common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -80,7 +80,7 @@ interface State {
 }
 
 interface Props {
-  location; //TODO: update type
+  location; //TODO: fixtype
```

Comment:
I've added types for [history](https://www.npmjs.com/package/@types/history), but I'm a bit confused how I'm actually supposed to use them here. Any help would be much appreciated!